### ISSUE TITLE
Could not compile dependency :xxh3 on Erlang/OTP 26

### ISSUE THREAD

**Issue by quentin-bettoum** [farhadi/xxh3#2]
Hello, I think there is a problem with Erlang/OTP 26. I can’t compile xxh3 dependency anymore, I get the following error.
`thread 'main' panicked at 'The NIF version from Erlang is not supported by Rustler: 2.17'`

According to https://github.com/rusterlium/rustler/issues/544, Rustler should be updated to 0.29

This is what I tried in local
```elixir
# use local version of xxh3 in mix.exs
  defp deps do
    [
    # ...
      {:xxh3, "~> 0.3.3", path: "/path/to/local/xxh3"}
    ]
  end
```

In  `native/xxh3/Cargo.toml`
```toml
[dependencies]
rustler = "0.29.1"
```

And now the dependency compiles, but I get this error when trying to call :xxh3
`Could not load module :xxh3, got: on_load_failure`

**Comment by farhadi** [comment]
Thanks for reporting the issue.
I just updated rustler to 0.29.1 and released xxh3 0.3.4 on hex.pm. Could you please give it a try with a clean _build directory?

**Comment by quentin-bettoum** [comment]
Hi,

It compiles now, but I still get the `on_load_failure` error. This is probably an issue with my current installation.
Here is the full error:
```elixir
14:08:40.557 [warning] The on_load function for module xxh3 returned:
{:error,
 {:load_failed,
  ~c"Failed to load NIF library: '/home/quentin/Dev/test_xxh3/_build/dev/lib/xxh3/priv/undefined.so: cannot open shared object file: No such file or directory'"}}
```

I will continue to investigate

**Comment by farhadi** [comment]
It looks the recent changes in rebar3_cargo plugin broke the load_nif_from_crate macro. I fixed it and published a new version 0.3.5. Please give it another try.

**Comment by quentin-bettoum** [comment]
It is working now, thanks a lot.
